mesonbuild/dependencies/framework.py

```diff
@@ -70,12 +77,13 @@ def detect(self, name: str, paths: T.List[str]) -> None:
             self.is_found = True
             return
 
-    def _get_framework_path(self, path: str, name: str) -> T.Optional[Path]:
+    def _get_framework_path(self, path: str, name: str) -> T.Optional[tuple[Path, str]]:
         p = Path(path)
         lname = name.lower()
         for d in p.glob('*.framework/'):
-            if lname == d.name.rsplit('.', 1)[0].lower():
-                return d
+            framework_name = d.name.rsplit('.', 1)[0]
```

Comment:
wouldn't be better to use `d.stem` here ?

Reply:
I implemented your other suggestion, so this change could be dropped completely, but I converted it to `stem` in my update. Let me know if I should drop it to keep the size of the diff down.